LICENSES_bundled.txt

```diff
@@ -3,22 +3,22 @@ compatibly licensed.  We list these here.
 
 Name: Numpydoc
 Files: doc/sphinxext/numpydoc/*
-License: 2-clause BSD
+License: BSD-2-clause
   For details, see doc/sphinxext/LICENSE.txt
 
 Name: scipy-sphinx-theme
 Files: doc/scipy-sphinx-theme/*
-License: 3-clause BSD, PSF and Apache 2.0
+License: BSD-3-clause AND PSF-2.0 AND Apache-2.0
```

Comment:
Out of curiosity, why the changed use of "AND"?

Reply:
Interesting: https://spdx.github.io/spdx-spec/appendix-V-using-SPDX-short-identifiers-in-source-files/#representing-multiple-licenses

Reply:
The specs seem to spell it `BSD-3-Clause` with a capital c, but I guess that just doesn't matter, if the auto-tool doesn't complain.

Reply:
@seberg Case corrected with https://github.com/numpy/numpy/pull/17238/commits/b769d8319532464652aafa78189a07df3252f478

Also note: I did not introduce change to the format of that file which is essentially a Debian machine readable copyright file, hence why I did not use SPDX-License-Identifier

Reply:
@charris you wrote:

> Out of curiosity, why the changed use of "AND"?

because the three licenses apply there https://github.com/scipy/scipy-sphinx-theme/blob/59c27fdde3ec4c78eb93b5e375d203ec5707382a/LICENSE.txt